frontend/ui/src/app/types/subscription.ts

```diff
@@ -0,0 +1,31 @@
+export type SubscriptionType = 'starter' | 'pro' | 'enterprise' | 'trial';
+
+export interface SubscriptionLimits {
+  maxCustomerOrganizations: number;
+  maxUsersPerCustomerOrganization: number;
+  maxDeploymentsPerCustomerOrganization: number;
+}
+
+export interface SubscriptionInfo {
+  subscriptionType: SubscriptionType;
+  subscriptionEndsAt: string;
+  subscriptionExternalId?: string;
+  subscriptionCustomerOrganizationQuantity?: number;
+  subscriptionUserAccountQuantity?: number;
+  currentUserAccountCount: number;
+  currentCustomerOrganizationCount: number;
+  currentMaxUsersPerCustomer: number;
+  currentMaxDeploymentTargetsPerCustomer: number;
+  trialLimits: SubscriptionLimits;
+  starterLimits: SubscriptionLimits;
+  proLimits: SubscriptionLimits;
+  enterpriseLimits: SubscriptionLimits;
```

Comment:
I don't think it's a bad idea to get this info from the backend, but we now have them twice. Limits can also be found in `deployment-targets.component.ts`, `users.component.ts`. They should probably be removed in favor of this